Dockerfile.base

```diff
@@ -18,7 +18,8 @@ COPY setup.py /root/LocalEGA/setup.py
 COPY lega /root/LocalEGA/lega
 COPY requirements.txt /root/LocalEGA/requirements.txt
 
-RUN pip install -r /root/LocalEGA/requirements.txt
+RUN pip install --upgrade pip && \
```

Comment:
....because....🥁🥁🥁

Reply:
because:
1. good practice to separate application specific requirements from environment
2. easier to see where that specific layer fails if a certain dependency is broken
3. all the cool image providers do it like that :) -- this is not a reason but seems to be somewhat of a convetion

Reply:
Ok, I still don't see the point, but I'll update it into 2 `RUN` commands.